fgd/brush/func/func_button.fgd

```diff
@@ -18,8 +18,10 @@
 		1: "Don't move" : 0
 		32: "Toggle" : 0
 		256: "Touch Activates" : 0
-		512: "Damage Activates" : 0
-		1024: "Use Activates" : 1
+		512: "Damage Activates" : 0 [-MOMENTUM]
+		512: "Damage Activates" : 1 [+MOMENTUM]
+		1024: "Use Activates" : 1 [-MOMENTUM]
+		1024: "Use Activates" : 0 [+MOMENTUM]
```

Comment:
Are they mutually exclusive? We still want use buttons to work, bhop maps use them for stuff.

Reply:
Generally an individual button is going to want one or the other, not both. Also generally momentum buttons tend to use damage more than use so this being default makes the most sense to me. It's always changeable for +use buttons anyways, this doesn't break backwards compatibility or anything

Reply:
Not really sure on a consensus here - it's good for jump but maybe bad for bhop? Maybe best to leave both on or both off for impartiality?

Reply:
Leave use on, but make the damage one the momentum conditional

Reply:
That's good yeah, the use wont actually effect jump at all anyways.